### PR TITLE
[Deployments] Update service link configs for K8s yaml

### PR DESCRIPTION
**Pull request by yanmo96**

### CODE REVIEW

File: kubernetes/services/api-gateway.yaml [hunk 1]
```diff
@@ -43,7 +43,7 @@ data:
     server.max-initial-line-length=65536
 
     #refer to https://github.com/opentracing-contrib/java-spring-jaeger/tree/release-3.2.1
-    opentracing.jaeger.udp-sender.host=localhost
+    opentracing.jaeger.udp-sender.host=10.111.132.59
```
Comment (cj-chung) [inline kubernetes/services/api-gateway.yaml:46]: @yanmo96 Let's remain the `localhost` in the master, and leave the modification during the deployment phase. In that way, we can modify the IP address according to the cluster's location (medina or kolla, or others).

**Review by cj-chung — CHANGES_REQUESTED** [review]

File: kubernetes/services/api-gateway.yaml (hunk 1 above)
Comment (xieus) [reply kubernetes/services/api-gateway.yaml:46]: +1

File: kubernetes/services/dpm_manager.yaml [hunk 2]
```diff
@@ -35,12 +35,14 @@ data:
     group.topic.to.multicast.topic.map=multicast-topic1:group-topic1,group-topic3 multicast-topic2:group-topic2,group-topic4
 
     zetaGateway.enabled=false
+    zetaGateway.node.mac=e0:97:96:02:45:53
+    microservices.node.service.url=http://nodemanager-service.default.svc.cluster.local:9007/nodes
     microservices.zeta.management.url=http://10.213.43.90
-    microservices.gateway.service.url=http://localhost:9015
+    microservices.gateway.service.url=http://gatewaymanager-service.default.svc.cluster.local:9015
```
Comment (xieus) [inline kubernetes/services/dpm_manager.yaml:41]: This is a good change.

File: services/elastic_ip_manager/src/main/resources/application.properties [hunk 3]
```diff
@@ -47,7 +47,7 @@ management.health.redis.enabled=false
 rbac.policy.type=Enforced
 
 ##### Jaeger ####
-opentracing.jaeger.udp-sender.host=localhost
+opentracing.jaeger.udp-sender.host=10.111.132.59
```
Comment (xieus) [inline services/elastic_ip_manager/src/main/resources/application.properties:50]: Remove all "^M"

File: kubernetes/services/api-gateway.yaml (hunk 1 above)
Comment (yanmo96) [reply kubernetes/services/api-gateway.yaml]: Thanks will put them back

File: kubernetes/services/dpm_manager.yaml (hunk 2 above)
Comment (yanmo96) [reply kubernetes/services/dpm_manager.yaml:41]: Thanks

**Comment by yanmo96** [comment]
Changes are made according to the above comments. Please take a look, thanks

File: services/elastic_ip_manager/src/main/resources/application.properties (hunk 3 above)
Comment (xieus) [reply services/elastic_ip_manager/src/main/resources/application.properties]: Do you notice the ending character of each line, which is "^M"? Could you remove them.

Comment (yanmo96) [reply services/elastic_ip_manager/src/main/resources/application.properties]: Yes, in the latest file changes the "^M" are been removed

Comment (xieus) [reply services/elastic_ip_manager/src/main/resources/application.properties]: Can you check https://github.com/futurewei-cloud/alcor/pull/675/files, which shows that there was no  change to this application.properties?

Comment (yanmo96) [reply services/elastic_ip_manager/src/main/resources/application.properties]: I changed it back from "10.111.132.59" to "localhost" (the original code). Overall there is no change to the application.properties file :)

**Comment by yanmo96** [comment]
Please don't merge this PR yet, may still need more changes. Sorry about this

**Review by cj-chung — APPROVED** [review]
LGTM

File: services/elastic_ip_manager/src/main/resources/application.properties (hunk 3 above)
Comment (xieus) [reply services/elastic_ip_manager/src/main/resources/application.properties]: This remains unresolved.

Comment (yanmo96) [reply services/elastic_ip_manager/src/main/resources/application.properties]: Sorry about the miss understanding. Should be fixed now.

**Pull request merged**